loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/CheckFlowsTool.java

```diff
@@ -0,0 +1,129 @@
+
+import com.google.auto.service.AutoService;
+
+import eu.itesla_project.commons.tools.Command;
+import eu.itesla_project.commons.tools.Tool;
+import eu.itesla_project.computation.ComputationManager;
+import eu.itesla_project.computation.local.LocalComputationManager;
+import eu.itesla_project.iidm.import_.Importers;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.StateManager;
+import eu.itesla_project.loadflow.api.LoadFlow;
+import eu.itesla_project.loadflow.api.LoadFlowParameters;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+@AutoService(Tool.class)
+public class CheckFlowsTool implements Tool {
+    
+    private CheckFlowsConfig checkFlowsConfig;
+
+    private static Command COMMAND = new Command() {
+
+        @Override
+        public String getName() {
+            return "check-flows";
+        }
+
+        @Override
+        public String getTheme() {
+            return "Computation";
+        }
+
+        @Override
+        public String getDescription() {
+            return "Check flows of a network";
+        }
+
+        @Override
+        public Options getOptions() {
+            Options options = new Options();
+            options.addOption(Option.builder().longOpt("case-file")
+                    .desc("case file path")
+                    .hasArg()
+                    .argName("FILE")
+                    .required()
+                    .build());
+            options.addOption(Option.builder().longOpt("output-file")
+                    .desc("output file path")
+                    .hasArg()
+                    .argName("FILE")
+                    .required()
+                    .build());
+            options.addOption(Option.builder().longOpt("load-flow")
+                    .desc("run loadflow")
+                    .build());
+            options.addOption(Option.builder().longOpt("verbose")
+                    .desc("verbose output (true/false)")
+                    .hasArg()
+                    .argName("TRUE/FALSE")
+                    .build());
+            return options;
+        }
+
+        @Override
+        public String getUsageFooter() {
+            return null;
+        }
+
+    };
+
+    @Override
+    public Command getCommand() {
+        return COMMAND;
+    }
+
+    @Override
+    public void run(CommandLine line) throws Exception {
+        Path caseFile = Paths.get(line.getOptionValue("case-file"));
+        Path outputFile = Paths.get(line.getOptionValue("output-file"));
+        CheckFlowsConfig config = new CheckFlowsConfig(getConfig().getThreshold(),
+                                                       line.hasOption("verbose") ? Boolean.parseBoolean(line.getOptionValue("verbose")) : getConfig().isVerbose(),
+                                                       getConfig().getLoadFlowFactory(),
+                                                       getConfig().getTableFormatterFactory());
```

Comment:
CheckFlowsConfig config = CheckFlowsConfig.load();
config.setVerbose(line.hasOption("verbose");

To be consistent, add all the setters to the CheckFlowsConfig class